README.md

```diff
@@ -4,7 +4,7 @@
 
 ## Overview
 
-Provide pagination using a `Link` HTTP header as described in [Github's developer documentation](github-pagination)
+Provide pagination using a `Link` HTTP header as described in [Github's developer documentation](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#pagination)
```

Comment:
Thanks for the contribution! Instead of changing it here, would you please change it on line 86 and then change the parentheses around "github-pagination" to square brackets. That's what was causing the broken link.

Reply:
Thanks for the review and for the library. Didn't notice the brackets problem at first :)

Reply:
I also changed the title of the link since GitHub's changed the title of the page. Let me know if you'd like to keep the original title.